package.json

```diff
@@ -62,25 +62,25 @@
 	"devDependencies": {
 		"@commitlint/cli": "^8.3.5",
 		"@commitlint/config-conventional": "^8.3.4",
-		"@formatjs/intl-pluralrules": "^1.5.3",
-		"@formatjs/intl-relativetimeformat": "^4.5.10",
+		"@formatjs/intl-pluralrules": "^2.2.6",
+		"@formatjs/intl-relativetimeformat": "^5.2.6",
```

Comment:
These are major versions. Are there any breaking changes we need to be aware of?

Reply:
Not they just drop IE11 support. Should have written that somewhere.